tutorials/notebooks/example_keras_nanodet_plus.ipynb

```diff
@@ -266,30 +266,31 @@
    "outputs": [],
    "source": [
     "import model_compression_toolkit as mct\n",
+    "from typing import Iterator, Tuple, List\n",
     "\n",
-    "TRAIN_DATASET_FOLDER = '/content/coco/val2017'\n",
-    "TRAIN_DATASET_ANNOTATION_FILE = '/content/coco/annotations/instances_val2017.json'\n",
+    "REPRESENTATIVE_DATASET_FOLDER = '/content/coco/val2017'\n",
+    "REPRESENTATIVE_DATASET_ANNOTATION_FILE = '/content/coco/annotations/instances_val2017.json'\n",
     "n_iters = 20\n",
     "\n",
-    "# Load COCO train set\n",
-    "train_dataset = coco_dataset_generator(dataset_folder=TRAIN_DATASET_FOLDER,\n",
-    "                                       annotation_file=TRAIN_DATASET_ANNOTATION_FILE,\n",
-    "                                       preprocess=nanodet_preprocess,\n",
-    "                                       batch_size=BATCH_SIZE)\n",
+    "# Load COCO representative dataset\n",
+    "representative_dataset = coco_dataset_generator(dataset_folder=REPRESENTATIVE_DATASET_FOLDER,\n",
+    "                                                annotation_file=REPRESENTATIVE_DATASET_ANNOTATION_FILE,\n",
+    "                                                preprocess=nanodet_preprocess,\n",
+    "                                                batch_size=BATCH_SIZE)\n",
     "\n",
     "# Define representative dataset generator\n",
-    "def get_representative_dataset(n_iter, train_loader):\n",
-    "\n",
-    "    def representative_dataset():\n",
-    "        ds_iter = iter(train_loader)\n",
+    "def get_representative_dataset(n_iter: int, dataset_loader: Iterator[Tuple]):\n",
+    "    \n",
+    "    def representative_dataset() -> Iterator[List]:\n",
```

Comment:
Need function doc:

>  
       """
        Creates a representative dataset generator from a PyTorch data loader, The generator yields numpy
        arrays of batches of shape: [Batch, H, W ,C]
        Returns:
            A representative dataset generator
        """